#!/usr/bin/env python
PACKAGE = 'radiation_utils'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("points_scale", double_t, 0, "Points scale", 0.02, 0.005, 1.0)
gen.add("lines_scale", double_t, 0, "Lines scale", 0.04, 0.005, 0.15)

exit(gen.generate(PACKAGE, "radiation_utils", "batch_visualizer"))
